app/controllers/base_public_controller.rb

```diff
@@ -2,13 +2,14 @@ class BasePublicController < ApplicationController
   include ClaimSessionTimeout
 
   helper_method :current_policy, :current_policy_routing_name, :claim_timeout_in_minutes
+  before_action :add_view_paths
   before_action :end_expired_claim_sessions
   after_action :update_last_seen_at
 
   private
 
   def current_policy
```

Comment:
I think we would not want there to be a single `current_policy` but rather multiple `current_policies` - and, given the introduction of `Journey` we probably want this to be `current_journey` instead.